exonum-java-binding-core/src/test/java/com/exonum/binding/storage/database/ForkTest.java

```diff
@@ -31,22 +28,16 @@
 
 @PrepareForTest({
     Views.class,
-    ViewModificationCounter.class,
 })
 @Disabled
-// TODO Won't run on Java 10 till Powermock is updated [ECR-1614].
+// TODO Won't run on Junit 5 till Powermock is updated [ECR-1614].
```

Comment:
to be more precise, till Powermock gains support of JUnit 5 (we use the most recent version)